tests/sentry/api/bases/test_organization.py

```diff
@@ -326,60 +326,9 @@ def test_invalid_params(self):
             self.run_test([self.env_1, self.env_2], ["fake", self.env_2.name])
 
 
-class GetTeamsTest(BaseOrganizationEndpointTest):
```

Comment:
Should we write similar tests for the get_snuba_params get teams?

Reply:
Don't have anything that explicitly testing `get_snuba_params` today, opted for adding a test with no teams specified and asserting that the `team_key_transactions` are only for my teams.